### PR TITLE
FisherChen3 add waltham

### PR DESCRIPTION
**Pull request by FisherChen3**
This pull request is to add waltham to the list of hometowns

cross reference to the issues
#21

### CODE REVIEW

File: FisherChen3-Waltham.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+### Attractions / Places to see
+
+- attr1
+- attr2
+
+### Fun Nearby
+
+- beach
+- lake
```
Comment (ROlety) [inline FisherChen3-Waltham.md]: Any particular good lake to see fall colors around the lake in fall?

Comment (ROlety) [inline FisherChen3-Waltham.md]: Testing my previous comment

Comment (FisherChen3) [inline FisherChen3-Waltham.md]: done!

Comment (FisherChen3) [inline FisherChen3-Waltham.md]: done

Comment (FisherChen3) [inline FisherChen3-Waltham.md]: @ROlety  can I get the approval?

**Comment by ROlety** [comment]
good

**Pull request merged**